packages/moonstream-components/src/components/LandingNavbar.js

```diff
@@ -122,6 +122,9 @@ const LandingNavbar = () => {
               colorScheme={
                 web3Provider.buttonText === web3Provider.WALLET_STATES.CONNECTED
                   ? "green"
+                  : web3Provider.WALLET_STATES.UNKNOWN_CHAIN ===
+                    web3Provider.buttonText
```

Comment:
Using a user-facing string to determine the wallet state feels very strange to me. Wouldn't it be more natural to store web3Provider.walletState as an enum?

Reply:
I removed `Unknown chain` as it is not used anymore, for the enum.  

Strings in enum might be not conventional, but they allow to get away without creating custom component for now. and I would prefer to track and change it in different PR. 

Correct solution would be to have react component `<MetamaskButton>` which will have logic of mapping state enum in to text. I propose to track it in a separate issue.

Reply:
Ok, np.